api-docs/api-operations/methods/patch-attach-configuration-group-to-ha-instance-version-accountid-ha-haid.rst

```diff
@@ -0,0 +1,312 @@
+
+.. _patch-attach-configuration-group-to-ha-instance-version-accountid-ha-haid.rst:
+
+Attach Configuration Group to HA Instance 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code::
+
+    PATCH /{version}/{accountId}/ha/{haId}
+
+Attaches a specified configuration group to the HA Instance (See :ref:`post-create-configuration-version-accountid-configurations`).
+
```

Comment:
Attaches a specified :ref:`configuration group <cdb-dg-concepts-configgroup>` to the HA Instance.